index.html

```diff
@@ -272,14 +272,14 @@ <h2>Scope</h2>
                 
                 <p>
                     As a result of this work, the Working Group will produce an updated version of EPUB 3,
-                    referred to as “EPUB 3.4” in this document. It is a primary goal of the new EPUB version to
+                    referred to as “EPUB 3.4” and “EPUB Accessibility 1.2” in this document. It is a primary goal of the new EPUB version to
```

Comment:
This sounds kind of strange. The new version of EPUB 3 is referred to as EPUB 3.4 in the charter. It's not referred to by both standard names.

A new version of the accessibility specification is something else as we don't tie it exclusively to EPUB 3.X.

Maybe this should be more like:

As a result, the Working Group will produce an updated version of EPUB 3 and its accessibility guidelines, referred to as "EPUB 3.4" and "EPUB Accessibility 1.2", respectively, in this document.

Reply:
You are right. I will change it.